decisions/0007-bulk-operations.rst

```diff
@@ -0,0 +1,48 @@
+7. Bulk Course and Course Run Operations
+=========================================
+
+Status
+-------
+Accepted (April, 2025)
+
+
+Context
+--------
+Publisher MFE is the entry point for Course Authors, Project Coordinators, and Course Editors to manage courses and course runs. Any action performed on publisher MFE results in calling course-discovery APIs.
+course-discovery is the backend service, primarily acting as product catalog and data aggregator. discovery maintains the communication and information exchange with platform and ecommerce.
+
+Due to this, when actions are performed on Publisher, the appropriate side-effects and data push to relevant systems are handled automatically. For example, creating the course and course run from publisher ends up creating the course run in Studio.
+This eliminates the additional step of setting up the course on Studio manually.
```

Comment:
nit: in Studio

Reply:
I checked with Grammarly, `This eliminates the additional step of setting up the course on Studio manually.` is correct.